poandy/controller/trade.py

```diff
@@ -0,0 +1,45 @@
+from poandy.util.request import RequestSender, RequestType
+from poandy.controller.base import Controller
+
+
+class TradeController(Controller):
+    @classmethod
+    def get_trades(cls, account_id, state="OPEN", instrument=None, count=50):
```

Comment:
I think we can also add in the params for ids and BeforeID

Reply:
It is possible, but I was wondering how to write tests for those, because it depends on the actual orders placed on the Oanda system. I was thinking about mocking but I have never done mocking before, but I can try explore it.

Reply:
Yes, mocking may be difficult. But for now, manually testing it should do. It's not easy to implement automated testing for every feature now.

Reply:
https://mockito-python.readthedocs.io/en/latest/ just good to know. i think you can just write down your observation as commetns